### ISSUE TITLE
most recent crashes iphone

### ISSUE THREAD

**Issue by UseDotCom** [UseDotCom/lighthouse#534]
Created by **[usedotcom](https://github.com/usedotcom)**  at 14:44Z on 05/22/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

Most recent crashes iphone and may require restart

- - - -

Lighthouse ticket URL: http://use-com.lighthouseapp.com/projects/103799/tickets/115

Was assigned to **[usedotcom](https://github.com/usedotcom)**

**Comment by UseDotCom** [comment]
By **Dave** at 19:41Z on 05/22/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

Can you provide some more details:

1) Which iphone
2) Which version of iOS?
3) Any specific account?
4) Which browser (Safari? Chrome?)

I tried this on my phone and everything worked.

**Comment by UseDotCom** [comment]
By **Dave** at 19:41Z on 05/22/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

Also, I tried this on the iOS emulator and it worked there as well.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 20:54Z on 05/22/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

Steve, Jonathan, and I are all experiencing this issue in Safari. We have cleared our caches and still happens. We are using 4, 4s, and 5. All have updated versions. Mine(iphone 4) is 6.1.3, Jonathan's(iphone 4s) is 6.1.2

When viewing Most Recent, the page partially loads then freezes. Steve said last time he tried it, it did not hard lock but locked up for 10-15 seconds - showed blank screen until it could refresh.

This may or may not be helpful. 

https://github.com/paulirish/infinite-scroll/issues/3

**Comment by UseDotCom** [comment]
By **Dave** at 22:52Z on 05/22/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

After looking a bit, it doesn't appear that we're using infinite scroll.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 03:20Z on 05/23/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

I thought we were. Guess my info was wrong. Sorry about that.

**Comment by UseDotCom** [comment]
By **Dave** at 03:47Z on 08/22/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

After a couple of days of playing with this, I think I have a fix for this. Nothing obvious stuck out and in the process I changed this from using the jQuery 'get' method to using the 'ajax' method. Things seem responsive now. Note that I fixed this on usetrader so there is no need to transfer the files there.

File:
templates/search3.txt.tmpl

**Comment by UseDotCom** [comment]
By **Dave** at 21:44Z on 08/30/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

I know this is on hold, but I had a chance to swing by Best Buy and I found that some iDevices had the issue, others did not. Ex: iPad Mini had no problems whatsoever. iPod touch exhibited the behavior above. That being the case, whenever we want to take this ticket off hold I'll have something to work with.

**Comment by UseDotCom** [comment]
By **Dave** at 01:35Z on 09/16/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/115))*

- - - -

Ok, I tracked down the issue. There's a variable that is caching references to all of the image elements. I've recoded the page to work without this. This seems to have corrected the issue, at least on the iPod Touch I'm using on iOS 6. Note that the fix was coded on usetrader.com and as such is already in place.

Files:

pub/search3.txt.tmpl
